src/test/java/org/candlepin/resource/EntitlementResourceTest.java

```diff
@@ -352,4 +355,103 @@ public void getAllEntitlementsForConsumer() {
         assertEquals(1, result.size());
         assertEquals("getAllEntitlementsForConsumer", result.get(0).getId());
     }
+
+    @Test
+    public void testUpdateEntitlementWithNullQuantity()
+    {
+        String entitlementId = "uniqueId";
+        Entitlement entitlement = new Entitlement();
+        entitlement.setId(entitlementId);
+        entitlement.setQuantity(3);
+
+        EntitlementDTO entitlementDTO = new EntitlementDTO();
+        entitlementDTO.setId(entitlementId);
+        entitlementDTO.setQuantity(null);
+
+        doReturn(entitlement).when(entitlementCurator).get(eq(entitlementId));
+
+        entResource.updateEntitlement(entitlementId, entitlementDTO);
+
+        verify(entitler, never()).adjustEntitlementQuantity(any(Consumer.class), any(Entitlement.class), any(Integer.class));
+    }
+    @Test
```

Comment:
Not sure if checkstyle will catch it, but separate your methods/blocks with whitespace wherever possible.

Reply:
Added whitespace to separate